recipes/pistache/all/patches/0001-remove-fpic.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 9cdac6b..b2d13b4 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -12,7 +12,6 @@ set(SOURCE_FILES
+ )
+ 
+ add_library(pistache OBJECT ${SOURCE_FILES})
+-set_target_properties(pistache PROPERTIES POSITION_INDEPENDENT_CODE 1)
```

Comment:
If the project is responsive, maybe add an `PISTACHE_PIC` option that sets this property dynamically as:
```cmake
option(PISTACHE_PIC "Enable pistache PIC" ON)
set_target_properties(pistache PROPERTIES POSITION_INDEPENDENT_CODE $<BOOL:${PISTACHE_PIC}>)
```

If you're willing to create a pr...

Reply:
Done: https://github.com/pistacheio/pistache/pull/855